applications/nutanix-ai/2.5.0/.bloodhound.yaml

```
@@ -0,0 +1,30 @@
+# Kubernetes versions
+# (e.g. 1.33.0 - has to match versions in https://github.com/yannh/kubernetes-json-schema)
+k8s_versions:
+  - 1.33.0
```

was it intentional to avoid 1.34.x ? 2.16 apps need to be compatible with 2.17 k8s version as applications will be upgraded before infra upgrade **usually**
```suggestion
  - 1.33.0
  - 1.34.0
```

---

I guess by mistake, added back